src/amber/cli/templates/app/.amber.yml.ecr

```diff
@@ -2,3 +2,27 @@ type: app
 database: <%= @database %>
 language: <%= @language %>
 model: <%= @model %>
+watch:
+  server: # required: the first command for this task is blocking
+    files:
+      - "src/**/*.cr"
+      - "src/**/*.<%= @language %>"
+      - "config/**/*.cr"
+    commands:
+      - "crystal build -o bin/<%= @name %> src/<%= @name %>.cr -p --no-color"
+      - "bin/<%= @name %>"
+
+  client: # optional: these files changes trigger browser reloading
+    files:
+      - "public/**/*"
+    commands: []
+
+  webpack: # optional: compiles assets using webpack
```

Comment:
I thought the new Amber watch was going to decouple webpack meaning you can choose which ever front end builder you want? Or is this a simple task?

Reply:
This is just a simple task, Webpack removal is addressed here: #866